IPython/lib/pretty.py

```diff
@@ -108,15 +108,7 @@ def _safe_getattr(obj, attr, default=None):
     except Exception:
         return default
 
-if PY3:
-    CUnicodeIO = StringIO
-else:
-    class CUnicodeIO(StringIO):
-        """StringIO that casts str to unicode on Python 2"""
-        def write(self, text):
-            return super(CUnicodeIO, self).write(
-                cast_unicode(text, encoding=get_stream_enc(sys.stdout)))
-
+CUnicodeIO = StringIO
```

Comment:
We may want to search for CUnicodeIO and replace instance by StringIO directly then deprecate CUnicodeIO